tests/i18n-gql.e2e.spec.ts

```diff
@@ -440,21 +439,21 @@ describe('i18n module e2e graphql', () => {
             path: ['validation'],
             extensions: {
               code: 'INTERNAL_SERVER_ERROR',
-              exception: {
-                response: 'Bad Request',
-                status: 400,
-                message: 'Bad Request',
-                name: 'I18nValidationException',
-                errors: [
-                  {
-                    property: 'age',
-                    children: [],
-                    constraints: {
-                      min: 'age with value: "2" needs to be at least 10, ow and COOL',
-                    },
-                  },
-                ],
-              },
+              // exception: {
```

Comment:
https://github.com/nestjs/graphql/compare/v10.2.0...v11.0.0
https://github.com/nestjs/graphql/issues/2676

https://github.com/nestjs/graphql/blob/7b017103ba66ebe310a9d7adb56c89e92a836494/packages/apollo/tests/e2e/pipes.spec.ts#L28-L38

Currently it does not appear to be possible to include error messages with class-validator in the response
I will send a report to `@nestjs/graphql`